core/src/test/java/org/bitcoinj/core/BloomFilterTest.java

```diff
@@ -59,7 +59,7 @@ public void insertSerializeTest() {
     
     @Test
     public void insertSerializeTestWithTweak() {
-        BloomFilter filter = new BloomFilter(3, 0.01, 2147483649L);
```

Comment:
Why not simply strip the `L`?

Reply:
Ah, probably the number is too large to fit an int and you didn't want to change the test data.